### DISCUSSION TITLE
Bootstrap out of date

### DISCUSSION THREAD

**Discussion by rmschindler** [day8/re-com#307] [Q&A] [answered]
Please pardon me if I simply don't understand, but I simply notice that it seems that Bootstrap v 3.2.0 is being used by Re-com, but the current version of Bootstrap is v 4.4.1. Why is this so? Is this intended (because Bootstrap is being deprecated anyway)? I am very curious.

Blessings,
Raphael

**Comment by sunng87** [comment]
I think we just need some effort to port these components to newer version of bootstrap. Now it's 5.x.

**Comment by superstructor** [comment, marked as answer]
See #284 

Bootstrap is being entirely removed. Upgrading it would be a significant job and wasted effort.
